examples/hello_world.rs

```diff
@@ -1,12 +1,12 @@
-type Exception = Box<dyn std::error::Error + Send + Sync + 'static>;
+use async_std::task;
 
-#[runtime::main]
-async fn main() -> Result<(), Exception> {
-    femme::start(log::LevelFilter::Info)?;
+fn main()  {
+    femme::start(log::LevelFilter::Info);
 
-    let uri = "https://httpbin.org/get";
-    let string = surf::get(uri).recv_string().await?;
-    println!("{}", string);
-
-    Ok(())
+    task::block_on(async {
+        let uri = "https://httpbin.org/get";
+        let string = surf::get(uri).recv_string().await;
+        println!("{}", string.unwrap());
+    });
+    ()
```

Comment:
Let's delete this line and the semicolon before it. The main function can still return `Result<(), Exception>` and we don't need the `.unwrap()` inside the `async` block.

Reply:
@stjepang `task::block_on` seems to always return `()`. I didn`t found a way of bubbling up the return :(

Reply:
I can remove the two lines and still maintain the `unwrap()` though :smile: